live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/repository/filter/AspectFilterApplier.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * #%L
+ * Alfresco HX Insight Connector
+ * %%
+ * Copyright (C) 2023 Alfresco Software Limited
```

Comment:
:warning: Alert! Wrong copyright year! :warning:

Reply:
Good catch - will fix that in the coming PR.